packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -230,7 +230,7 @@ export const Components = {
     card: (name: string) => `data-testid New transform ${name}`,
     Reduce: {
       modeLabel: 'Transform mode label',
```

Comment:
Should this be changed too?

Reply:
The e2e tests aren't actually attempting to select this label, but I think that's a good idea. I'll also add a test to verify.

Reply:
Ah okay, looks good then 👍